design/requirements.md

```diff
@@ -18,7 +18,29 @@ Discuss things we want to do, and things we *don't* want to do
 
 ### Definitions, acronyms, and abbreviations
 
-Anything not obvious
+**Work:** Any content a user has posted to Mixxy, typically an illustration.
+
+**Remix:** The act of one user making a copy of another's work for the purpose of editing, improving upon, or otherwise modifying the original work.  (The original work is *not* modified in the process; remixing cannot be used for vandalism.)
+
+**Content:** The subject matter of a work.
```

Comment:
You know what, after the first draft is written, I'll go through and proofread it; things like this will be fixed